theme.json

```diff
@@ -0,0 +1,413 @@
+        }
+      },
+      "h2": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--4--font-size)",
+          "lineHeight": "var(--wp--custom--heading--4--line-height)",
+          "fontWeight": "var(--wp--custom--heading--4--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--4--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--4--letter-spacing)"
+        }
+      },
+      "h3": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--3--font-size)",
+          "lineHeight": "var(--wp--custom--heading--3--line-height)",
+          "fontWeight": "var(--wp--custom--heading--3--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--3--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--3--letter-spacing)"
+        }
+      },
+      "h4": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--2--font-size)",
+          "lineHeight": "var(--wp--custom--heading--2--line-height)",
+          "fontWeight": "var(--wp--custom--heading--2--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--2--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--2--letter-spacing)"
+        }
+      },
+      "h5": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "h6": {
+        "typography": {
+          "fontSize": "var(--wp--custom--heading--1--font-size)",
+          "lineHeight": "var(--wp--custom--heading--1--line-height)",
+          "fontWeight": "var(--wp--custom--heading--1--font-weight)",
+          "fontFamily": "var(--wp--custom--heading--1--font-family)",
+          "letterSpacing": "var(--wp--custom--heading--1--letter-spacing)"
+        }
+      },
+      "button": {
+        "color": {
+          "background": "var(--wp--custom--button--primary--default)",
+          "text": "var(--wp--custom--button--primary--label)"
+        },
+        "typography": {
+          "fontSize": "clamp(1.00rem, calc(1.16rem + -0.15vw), 1.13rem)",
+          "lineHeight": "var(--wp--custom--body--2--line-height)",
+          "fontWeight": "var(--wp--custom--body--2--font-weight)",
+          "fontFamily": "var(--wp--custom--body--2--font-family)"
+        },
+        "spacing": {
+          "padding": {
+            "top": "clamp(0.50rem, calc(0.81rem + -0.30vw), 0.75rem)",
+            "bottom": "clamp(0.50rem, calc(0.81rem + -0.30vw), 0.75rem)",
+            "left": "1.50rem",
+            "right": "1.50rem"
+          }
+        },
+        "border": {
+          "radius": "999999px"
```

Comment:
Is there a cleaner way to round a whole element off besides doing this? this is what I see everyone doing but it always feels hacky to me.

Reply:
That's why I have noticed as well, so I left that as is

Reply:
I'm gonna make this a token which says round so it's easily changeable